Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM python:2.7-onbuild
+FROM python:3.7
+
+WORKDIR /usr/src/app
```

Comment:
The first line `WORKDIR /usr/src/app` technically isn't required, but is more explicit and in my opinion makes it slightly easier to understand what is going on. 

The `COPY . .` line is required because I changed the container from an ON-Build style container (which was not an official container btw) to the official python 3.7 container. This requires that you explicitly copy the working directory into the container.

Reply:
Great, thanks for the PR!